resources/zh/cemu.po

```diff
@@ -1051,6 +1083,19 @@ msgstr "垂直同步"
 msgid "Controls the vsync state"
 msgstr "控制垂直同步状态"
 
+#: GeneralSettings2.cpp
+msgid "Async shader compile"
+msgstr "异步着色器编译"
+
+#: GeneralSettings2.cpp
+msgid ""
+"Enables async shader and pipeline compilation. Reduces stutter at the cost "
+"of objects not rendering for a short time.\n"
+"Vulkan only"
+msgstr ""
+"启用异步着色器和流水线编译。可以减少对象在短时间内无法渲染时的卡顿。\n"
```

Comment:
DeepL translated it back to English as "[...] This can reduce stuttering when objects fail to render for a short period of time."
If that's correct, there is an interpretation error in your translation. The meaning of the original string is that async compilation error causes objects not to render for a short period of time, in order to reduce stuttering (which is caused by _synchronous_ shader compilation). In other words, it delays rendering of objects to reduce stuttering.